### DISCUSSION TITLE
Two versions of Jamulus

### DISCUSSION THREAD

**Discussion by ghost** [jamulussoftware/jamulus#1227] [UI Related]
I can see that there are users and developers that are willing to expand (and add code, much code) Jamulus more than the KISS principle. The KISS version should be named Jamulus, and the luxury version could be named Jamulux. Jamulus will be made better by removing more code than it adds after 3.7.0. And Jamulux will be allowed to grow without bound (like most software).

**Comment by gene96817** [comment]
I think it is much better to have one Jamulus version with KISS as the default. Necessary extra features should be made available without confusing new users. In this way new users can develop experience and grow into the expert features. A single version makes support much easier (assuming we keep the UI well organized). 

There is no clear way to separate users into Jamulus or Jamulux. Invariably, there will be one feature a Jamulus user needs from Jamulux. This creates the scenario of leaving KISS behind and drowning in the Jamulux pool for just one more feature. I expect for every feature in Jamulux, there is a strong reason to need the feature. It is just that a new user doesn't have the experience to know why that (advanced) feature is needed.  There will be an infinite number of ways where a KISS user needs just one advanced feature. This makes defining Jamulus and Jamulux impossible.
